torchvision/prototype/transforms/_meta.py

```diff
@@ -18,6 +18,9 @@ def __init__(self, format: Union[str, features.BoundingBoxFormat]) -> None:
         self.format = format
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+        # datumbox: We shouldn't have if/elses here. We could just set:
+        # `_transformed_types = (features.BoundingBox,)`
+        # and eliminate them.
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
@pmeier can you include that into the same PR with `ToTensor` update

Reply:
I'll be picking this up.

Reply:
#6494